integration/test/API/Brig.hs

```diff
@@ -320,9 +319,9 @@ uploadKeyPackages cid kps = do
       "/mls/key-packages/self/" <> cid.client
   submit
     "POST"
-    ( req
+    do
+      req
         & addJSONObject ["key_packages" .= map (T.decodeUtf8 . Base64.encode) kps]
-    )
```

Comment:
i would say this change is noise; i'd prefer if things weren't moving so quickly, this kind of change always risks creating low-temperature edit wars that go unnoticed for a while.